tests/integration/enc/models/test_vgg.py

```diff
@@ -30,101 +30,104 @@ def vgg_multi_layer_encoder_loaders(vgg_archs):
 vgg = MODELS["vgg11_bn"]
 
 
-@pytest.mark.slow
-def test_vgg_pretrained(mocker):
-    state_dict = vgg(pretrained=False).state_dict()
-    mocker.patch(
-        mocks.make_mock_target(
-            "enc", "models", "vgg", "hub", "load_state_dict_from_url"
-        ),
-        return_value=state_dict,
-    )
-
-    model = vgg(pretrained=True)
-    ptu.assert_allclose(model.state_dict(), state_dict)
-
-
-def test_vgg_pretrained_num_classes_mismatch():
-    with pytest.raises(RuntimeError):
-        vgg(pretrained=True, num_classes=1001)
-
-
-@pytest.mark.slow
-def test_VGGMultiLayerEncoder_unknown_arch():
-    arch = "unknown"
-    with pytest.raises(ValueError):
-        enc.VGGMultiLayerEncoder(arch, pretrained=False, internal_preprocessing=False)
-
-
-@pytest.mark.slow
-def test_vgg_multi_layer_encoder_smoke(
-    subtests, vgg_archs, vgg_multi_layer_encoder_loaders,
-):
-    for arch, loader in zip(vgg_archs, vgg_multi_layer_encoder_loaders):
-        with subtests.test(arch=arch):
-            multi_layer_encoder = loader(pretrained=False)
-            assert isinstance(multi_layer_encoder, enc.vgg.VGGMultiLayerEncoder)
-
-            with subtests.test("repr"):
-                asserts.assert_property_in_repr(repr(multi_layer_encoder), "arch", arch)
-
-
-@pytest.mark.large_download
-@pytest.mark.slow
-@pytest.mark.flaky
-def test_vgg_multi_layer_encoder(
-    subtests, vgg_archs, vgg_multi_layer_encoder_loaders, enc_asset_loader
-):
-    for arch, loader in zip(vgg_archs, vgg_multi_layer_encoder_loaders):
-        with subtests.test(arch=arch):
-            asset = enc_asset_loader(arch)
-
-            multi_layer_encoder = loader(
-                pretrained=True,
-                weights="torch",
-                preprocessing=False,
-                allow_inplace=False,
+class TestVGGPretrained:
```

Comment:
This class should be about the VGG nets. Right now we only have tests that include the pretrained variant, but that should not be exclusive.

```suggestion
class TestVGG:
```